packages/flutter/lib/src/gestures/recognizer.dart

```diff
@@ -494,3 +499,49 @@ abstract class PrimaryPointerGestureRecognizer extends OneSequenceGestureRecogni
     properties.add(EnumProperty<GestureRecognizerState>('state', state));
   }
 }
+
+/// A container for a [local] and [global] [Offset].
+///
+/// Usually, the [global] [Offset] is in the coordinate space of the screen
+/// after conversion to logical pixels and the [local] offset is the same
+/// [Offset], but transformed to a local coordinate space.
+class CombinedOffset {
```

Comment:
I'm not a big fan of this name (it's not an Offset subclass)...

Maybe something based on the word Position? I don't have a good idea...

Reply:
Is there some way we can avoid making a public class at all for this?  It's not clear to me that it does anything outside of this use case we need for this class.

Reply:
Done.